### PR TITLE
Add logo and link to scala-js.org to README.md

### PR DESCRIPTION
**Pull request by reymon359**
Hi there! 👋🏽 

I found about this awesome project and seemed to me that the main `README.md` was missing a distinctive logo.

I added the one from the docs and a link to them.

Hope you like it and help other people to identify this repo better and makes it easier to access the docs 😊

### CODE REVIEW

**Comment by reymon359** [comment]
Seems like I missed the part of signing the Scala CLA to contribute to Scala.js 😓 

I already did and got this message, so I guess everything is fine right now.

Screenshot 2021-01-15 at 21 24 22

By the way, the GitHub action tells me to `comment on the pull request to ask for a new check.` so if someone could check it again to see if now passes it would be great. Thank you in advance! 😊

**Comment by gzm0** [comment]
@reymon359 thanks for your interest in contributing.

@sjrd, I think this is a nice touch, so I'm all for it. WDYT?

In any case, the commit message should probably be changed to `Add logo and link to scala-js.org to README.md.`. (we do not use prefixes other than `Fix #<issue>: `, commit title should start with uppercase, [read more](https://github.com/scala-js/scala-js/pull/4378)).

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+<p align="center">
+  <a href="http://www.scala-js.org/">
+    <img src="http://www.scala-js.org/assets/img/scala-js-logo.svg" height="128">
+    <h1 align="center">Scala.js</h1> 
+  </a>
+</p>
+
+
```
Comment (sjrd) [inline README.md:8]: Avoid double blank lines. Use only one blank line here.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+<p align="center">
```
Comment (sjrd) [inline README.md:1]: Given how readmes are rendered on GitHub, center alignment looks weird IMO. Consider aligning this, and the `<h1>`, on the left.

**Review by sjrd — CHANGES_REQUESTED** [review]
Yes, we can take this, why not?

I agree with the remark about the commit message. I also have two requested changes below.

**Comment by reymon359** [comment]
> @reymon359 thanks for your interest in contributing.
> 
> @sjrd, I think this is a nice touch, so I'm all for it. WDYT?
> 
> In any case, the commit message should probably be changed to `Add logo and link to scala-js.org to README.md.`. (we do not use prefixes other than `Fix #<issue>: `, commit title should start with uppercase, [read more](https://github.com/scala-js/scala-js/pull/4378)).

Hi there @gzm0, Thank you for the suggestion! I usually follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) standard,  will update the commit message then. 

By the way, the read more link you added redirects me to this same Pull Request in case you wanted to link it to some docs.

File: README.md (hunk 1 above)
Comment (reymon359) [reply README.md:8]: Thanks for the suggestion! I will update it 👌🏽

**Review by sjrd — APPROVED** [review]
Thank you :)

**Pull request merged**